deeplearning4j-keras/src/main/java/org/deeplearning4j/DeepLearning4jEntryPoint.java

```diff
@@ -0,0 +1,134 @@
+package org.deeplearning4j;
+
+import org.bytedeco.javacpp.FloatPointer;
+import org.bytedeco.javacpp.hdf5;
+import org.deeplearning4j.nn.modelimport.keras.InvalidKerasConfigurationException;
+import org.deeplearning4j.nn.modelimport.keras.KerasModelImport;
+import org.deeplearning4j.nn.modelimport.keras.UnsupportedKerasConfigurationException;
+import org.deeplearning4j.nn.multilayer.MultiLayerNetwork;
+import org.nd4j.linalg.api.ndarray.INDArray;
+import org.nd4j.linalg.factory.Nd4j;
+import org.nd4j.linalg.indexing.INDArrayIndex;
+import org.nd4j.linalg.indexing.NDArrayIndex;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+
+import static org.bytedeco.javacpp.hdf5.H5F_ACC_RDONLY;
+
+public class DeepLearning4jEntryPoint {
+
+    private static final Logger logger = LoggerFactory.getLogger(DeepLearning4jEntryPoint.class);
+
+    public DeepLearning4jEntryPoint() {
+
+    }
+
+    public void fit(
+            String modelFilePath,
+            String type,
+            String trainFeaturesFile,
+            String trainLabelsFile,
+            int batchSize,
```

Comment:
This is a lot. Maybe consider a POJO using lombok to generate the builder would be easier?